_posts/en/newsletters/2022-02-23-newsletter.md

```diff
@@ -0,0 +1,139 @@
+  efficient gossip protocol.
+
+- **CTV signet:** Jeremy Rubin [published][rubin ctv signet] parameters
+  and code for a [signet][topic signet] with
+  [OP_CHECKTEMPLATEVERIFY][topic op_checktemplateverify] activated.
+  This simplifies public experimentation with the proposed opcode and
+  makes it much easier to test compatibility between different software
+  using the code.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Will a post-subsidy block with no transactions include a coinbase transaction?]({{bse}}112193)
+  Pieter Wuille explains that since every block must have a coinbase transaction
+  and since every transaction must include at least one
+  input and one output, a post-subsidy block with no block reward (no fees and
+  no subsidy) will still require at least one zero-value output.
+
+- [How can the genesis block contain arbitrary data on it if the script is invalid?]({{bse}}112439)
+  Pieter Wuille lists the reasons why the genesis block's coinbase
+  "Chancellor..." text push is valid. First, the [genesis block][bitcoin se 13122] is valid by
+  definition. Second is that coinbase input scripts are not actually executed.
+  Third is that, pre-[BIP342][], the requirement of a single element on the stack
+  after execution is only a policy rule, not consensus. Finally, that policy
+  rule applies only when executing both input and output scripts and since the
+  genesis block's input does not have an output script, the policy does not
+  apply. Pieter also notes the reason for the genesis block's unspendability is
+  unrelated to this discussion and involves the original Bitcoin software [not
+  adding the genesis block][bitcoin github genesis] to its internal database.
+
+- [What is a Feeler Connection? When is it used?]({{bse}}112247)
+  User vnprc explains the purpose of Bitcoin Core's [feeler
+  connection][chaincode p2p] which is an outbound connection separate from the
+  default 8 outbound connections and 2 blocks-only outbound connections. The
+  feeler connection is used to test potential new peers suggested from the
+  gossip network as well as test previously unreachable peers and evict them accordingly.
+
+- [Are OP_RETURN transactions not stored in chainstate database?]({{bse}}112312)
+  Antoine Poinsot points out that since `OP_RETURN` outputs are
+  [unspendable][bitcoin github unspendable], they are not stored in the
+  [chainstate directory][bitcoin wiki data].
```

Comment:
Would the Bitcoin Core files doc be better than wiki? It is more accurate and closer to the source.
https://github.com/bitcoin/bitcoin/blob/master/doc/files.md#data-directory-layout